app/assess/routes.py

```diff
@@ -233,12 +233,11 @@ def landing():
     assessment_deadline = get_round(
         Config.COF_FUND_ID, Config.COF_ROUND2_ID
     ).assessment_deadline
-    
+
     # Updating assessment progress for applications
-    application_overviews = get_assessment_progress(
-        application_overviews
-        )
-  
+    if application_overviews:
+        application_overviews = get_assessment_progress(application_overviews)
```

Comment:
Bit confusing as looks like we're mutating this? Can we call it something different rather than mutate?

Reply:
Yeah good point.

Reply:
Addressed in https://github.com/communitiesuk/funding-service-design-assessment/pull/127/commits/d45232f3ee9c362c215d467060038b4ac9d9fb0d